docs/faqs.md

```diff
@@ -349,3 +338,33 @@ With version ranges the newest compatible package may yield a different package-
 - Build Reproducibility
 
 If consumers try to download and build the recipe at a later time, it may resolve to a different package version that may generate a different binary (that may or may not be compatible). In order to prevent these types of issues, we have decided to only allow exact requirements versions. This is a complicated issue, check [this thread](https://github.com/conan-io/conan-center-index/pull/9140#discussion_r795461547) for more. 
+
+## How to consume a graph of shared libraries?
+
+When the CI builds packages with `shared=True`, it applies the option only to the package being created, but not to
+the requirements. As the default value for the `shared` option is usually `False`, you can expect that the dynamic
+library that has just being generated has linked all its requirements as static libraries.
+
+It is important to remark the default [package id mode](https://docs.conan.io/en/latest/creating_packages/define_abi_compatibility.html#versioning-schema)
+used by Conan (which is the same default used by ConanCenter): `semver_direct_mode`. With this default only the major
+version of the requirements is encoded in the package ID. 
+
+The two previous behaviors together can lead to unexpected results for a user that want to consume a graph of
+dependencies as shared libraries from ConanCenter. They might think that using `*:shared=True` in their profile is
+enough, and indeed Conan will retrieve from ConanCenter all the dynamic libraries for all the graph of dependencies, but
+**all of them will contain the logic of their respective requirements embedded in the dynamic library**, and this
+logic is embedded at the time of building, so it might not match the version of the requirements that was resolved
+by Conan, and for sure, the other dynamic libraries won't be used, only the ones linked directly by the consumer
+project. See a more detailed [example here](https://github.com/conan-io/conan/issues/9712).
+
+In order to consume all those libraries as shared ones, building from sources is needed. This can be
+easily achievable using `*:shared=True` in the _host_ profile and `--build` in the install command. With these inputs,
+Conan will build from sources all the packages and use the shared libraries when linking.
+
+> Note.- The proper solution for recipes would be to use something like
```

Comment:
This might trick contributors to add it to the recipe. Given it's a blocker https://github.com/conan-io/conan-center-index/issues/8987#issuecomment-1022016332

I think the workaround above is more than enough

```suggestion
> ℹ️ Note: If you are hosting your own recipes, the proper solution for recipes would be to use something like
```